test/decap_out.bin

```diff
@@ -0,0 +1,3 @@
+6�W�����n���;������m� ĥ�B[H���#�Ӈ(��h�]:\�P��x�e���b��)G�f��"��˭f�	��J�)�����{�Hm�\Pú�+�P޸%��/jϙ%�؆�<_�~�
+K�JEh����lEa�:�(��/\Ѯ�b��î����-g,AY��4�
+l��t�N�)~\�HU4y�	}qJ���t#��}.��T���?��ϊ��cD=�L��nmv�{��ſԋȣ�
```

Comment:
Mark this and similar files as binary blobs in git (unless that's already the case), I would not expect to see the content presented as text for binary blobs.